packages/aws-cdk-lib/aws-ecs-patterns/README.md

```diff
@@ -1123,7 +1123,7 @@ const queueProcessingFargateService = new ecsPatterns.NetworkLoadBalancedFargate
 });
 ```
 
-### Use dualstack NLB
+### Use dualstack Load Balancer
```

Comment:
Can you have subheading with Network Load Balancer and Application Load Balancer and add some description please?

Reply:
I've updated docs. Could you please confirm it again?